oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -383,11 +384,6 @@ private APIGatewayProxyResponseEvent redirect(
                                             .getValue())
                             .claim("state", new State().getValue())
                             .claim("client_id", configurationService.getOrchestrationClientId())
-                            .claim(
-                                    "scope",
-                                    ValidScopes.extractAuthScopesFromRequestedScopes(
```

Comment:
Just a question really, but I can't see anywhere else this was being used. Do we still need the method?